packages/discovery-provider/src/tasks/entity_manager/entities/user.py

```diff
@@ -336,6 +336,15 @@ def update_user_metadata(
                 continue
             setattr(user_record, key, metadata[key])
 
+    # Ensure verified social handle is same as audius handle
```

Comment:
confused by this comment because it seems to be assigning rather than validating

Reply:
this is to prevent a verified twitter user to update his handle to be something else